lib/client-assets.php

```diff
@@ -268,6 +277,13 @@ function gutenberg_register_scripts_and_styles() {
 		filemtime( gutenberg_dir_path() . 'build/shortcode/index.js' ),
 		true
 	);
+	wp_register_script(
+		'wp-redux-routine',
+		gutenberg_url( 'build/redux-routine/index.js' ),
+		array(),
+		filemtime( gutenberg_dir_path() . 'build/redux-routine/index.js' ),
+		true
```

Comment:
Note: I passed this argument for consistency with how we register other scripts, but it's not obvious to me why we need `$in_footer` assigned for these registered scripts.